### ISSUE TITLE
Docs URL isn't distinguishable without relying on color

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#137]
The [Read the Docs URL](http://trix2.readthedocs.org/en/latest/student/gettingstarted.html) on Trix's frontpage isn't distinguishable without relying on color, a violation of [Success Criteria 1.4.3 in WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html):

> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.

![Screenshot from 2023-08-09 10-24-30](https://github.com/devilry/trix2/assets/796053/002dd8b6-92d4-4c1c-8c9c-66106b970876)

**Comment by Levijatan** [comment]
Added a link icon to the link in d466d5e
